tests/TestCase/View/Helper/PaginatorHelperTest.php

```diff
@@ -3495,4 +3495,35 @@ protected function setPagingParams($params, bool $merge = true)
         }
         $this->View->setRequest($this->View->getRequest()->withAttribute('paging', $params));
     }
+
+    /**
+     * Test that generated URLs work without sort defined within the request
+     *
+     * @return void
+     */
+    public function testPaginationWithPassedParams()
+    {
+        $request = new ServerRequest([
+            'url' => '/articles/index/whatever/3',
+            'params' => [
+                'plugin' => null,
+                'controller' => 'articles',
```

Comment:
Can this be Articles as well? Or does the test then fail?

Reply:
Refs https://github.com/cakephp/cakephp/pull/15568

Reply:
The test fails. The url result changes to capitalize. A similar test with the same problem [here].(https://github.com/cakephp/cakephp/blob/master/tests/TestCase/View/Helper/PaginatorHelperTest.php#L817)
I don´t know what's happening.

Reply:
OK; we can tackle this in a follow up PR, this one is fine then.